maintainers/maintainer-list.nix

```diff
@@ -9299,6 +9299,11 @@
     githubId = 303489;
     name = "Manuel Bärenz";
   };
+  tu-maurice = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add tu-maurice``.